service/http.directory/src/test/scripts/org/kaazing/gateway/service/http/directory/post.large.data.rpt

```diff
@@ -20,29 +20,12 @@
-write "\r\n"
-write notify BARRIER
+write method "POST"
```

Comment:
Test name is POST large data.  There is no data being posted in this script.  The wireshark before and after should be the same or very similar